packages/@jsii/go-runtime/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/aws-cdk/jsii
```

Comment:
actually should be `github.com/aws/jsii` aka, the repo. As far as I can tell yes, this can be the module path. Will need to experiment once publishable if the nesting (module not being in repo root) actually means that being published as is requires imports like `github.com/aws/jsii/packages/@jsii/go-runtime/jsii` but I'm not quite sure.